tests/test_cache.py

```diff
@@ -140,6 +140,8 @@ def test_abstract_methods(self):
             cache.clear_all()
         with self.assertRaises(NotImplementedError):
             cache.save(make_package())
+        with self.assertRaises(NotImplementedError):
+            cache.search({'name': [], 'summary': []}, 'or')
```

Comment:
I actually don't understand how this test is passing. Isn't there an implemented `search()` method on the ICache class?